backend/vault.go

```diff
@@ -9,12 +9,25 @@ import (
 	"github.com/hashicorp/vault/api"
 	log "github.com/sirupsen/logrus"
 	"github.com/tuenti/secrets-manager/errors"
+
+	"github.com/prometheus/client_golang/prometheus"
 )
 
 var logger *log.Logger
 
 const defaultSecretKey = "data"
 
+var (
+	tokenExpired = prometheus.NewGaugeVec(prometheus.GaugeOpts{
```

Comment:
as discussed, I think exposing ttl and ttlmax as two separate metrics would allow us to write alerts when expiration is imminent.

wdyt?